package.json

```
@@ -58,10 +58,10 @@
     "eslint": "eslint --fix '*.{js,json}' '{src,tests}/**/*.{ts,tsx}'",
     "eslint:ci": "eslint '*.{js,json}' '{src,tests}/**/*.{ts,tsx}'",
     "pretest": "tsc --noEmit",
-    "test": "yarn node --experimental-vm-modules $(yarn bin jest)",
-    "test:ci": "yarn node --experimental-vm-modules $(yarn bin jest)",
-    "test:dev": "yarn node --experimental-vm-modules $(yarn bin jest) --watch --no-coverage",
-    "test:coverage:watch": "yarn node --experimental-vm-modules $(yarn bin jest) --watch",
+    "test": "vitest",
+    "test:ci": "vitest",
+    "test:dev": "vitest --no-coverage",
+    "test:coverage:watch": "vitest --coverage --watch",
```

Let's simplify this. https://github.com/pmndrs/jotai/pull/1839#discussion_r1166148724
```suggestion
    "test": "vitest --ui --coverage",
    "test:ci": "vitest",
```